apps/web/src/layouts/ProtectedLayout.tsx

```
@@ -0,0 +1,39 @@
+export default function ProtectedLayout({
+                                            children,
+                                        }: {
+    children: React.ReactNode;
+}) {
+    const {data, isLoading} = useSession();
+    const {authenticatedUser} = useAuthStore();
+
+    if (isLoading) {
+        return (
+            <div className="flex flex-col h-screen justify-center items-center">
+                <h1 className="text-xl font-medium"></h1>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)